frontend/components/Comment.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react'
+import Link from 'next/link'
+import Form from '../components/Form'
+import { Consumer } from '../Context'
+class Comment extends Component {
+  state = { editValue: '', errors: [], textfieldOpen: false }
+  change = event => {
+    this.setState({ editValue: event.target.value })
+  }
+  componentDidMount() {
+    this.setState(prevState => ({
+      editValue: this.props.content
+    }))
+  }
+  render() {
+    return (
+      <React.Fragment>
+        <div>
+          {this.props.content}
+          <br />
+          {this.props.authorName}
+          <br />
+          {this.props.createdAt}{' '}
+          {this.props.canEditOrDelete ? (
```

Comment:
using `&&` would be cleaner

Reply:
Where do I use it?

Reply:
use something like `this.props. canEditOrDelete && <div>...</div>`

Reply:
Done.